src/presets.js

```diff
@@ -150,7 +150,7 @@ const getNestedChangeLog = (
 };
 
 const attachNestedChangelog = (upstreams, commit, callback) => {
-	async.each(
+	async.map(
```

Comment:
Changing this to `eachSeries` would have had the equivalent result but doing everything in serial would have slowed things down a bunch